src/service/resources-impl.js

```diff
@@ -1967,11 +1967,11 @@ export class Resources {
       if (resource.getState() == ResourceState.NOT_BUILT) {
         resource.whenBuilt().then(() => {
           this.measureAndScheduleIfAllowed_(resource, layout,
-              parentResource.getPriority());
+              parentResource.getLayoutPriority());
```

Comment:
Looks like priority is also used for preload in addition to layout. I'm okay with this change but another option is to just update the jsdoc to more clearly spell this out in custom-element.js and base-element.js.

Reply:
Hm? It's used for preloading, but specifically for layout in preloading.